packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js

```diff
@@ -95,13 +99,14 @@ function getPageDetails( page ) {
 
 export default function PageDetails( { id } ) {
 	const { record } = useEntityRecord( 'postType', 'page', id );
-
+	const { records: templates } = useEntityRecords(
+		'postType',
+		'wp_template',
+		{ per_page: -1 }
+	);
```

Comment:
While I like this helper hook, I think it does and returns way more data than we need here. There's nothing wrong with good old `getEntityRecords( 'postType', 'wp_template', { per_page: -1 } )` 😄